homeassistant/components/verisure.py

```diff
@@ -53,6 +54,8 @@
         vol.Optional(CONF_SMARTPLUGS, default=True): cv.boolean,
         vol.Optional(CONF_THERMOMETERS, default=True): cv.boolean,
         vol.Optional(CONF_SMARTCAM, default=True): cv.boolean,
+        vol.Optional(CONF_POLLING_RATE, default=1): vol.All(
+            vol.Coerce(int), vol.Range(min=1)),
```

Comment:
We usually use `cv.time_period`.

Reply:
That would allow the user to set seconds, right? Verisure seems to lock people out that poll too often so I rather see that a minute is minimum.

Reply:
We can set a minimum time too.